modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
 separate upgrade strategies and also are used to control the cadence of updates.
 Channels are tied to a minor version of {product-title}. For instance,
 {product-title} {product-version} channels will never include an upgrade to a
-4.3 release. This ensures administrators make an explicit decision to upgrade to
+4.5 release. This ensures administrators make an explicit decision to upgrade to
 the next minor version of {product-title}. Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version}, which includes the upgrade from the previous 4.3
+release, has three upgrade channels to choose from:
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+The upgrade channels contain two types of updates:
+
+.  General Availability Software (or GA) - These versions of {product-title} are
+fully supported and are considered production quality. You may upgrade to the general
+availability release from either of the fast and stable channels.
+
+.  Release Candidate Software (or RC) - These versions of {product-title} are
+representative of the eventual general availability release and are available only
+in the candidate-{product-version} channel. The release candidate will contain all
+the features of the product. You are allowed to upgrade from a release candidate to
+another release candidate and to upgrade from a previous minor version of
+{product-title} to the current release candidate. Release candidate builds are not
+supported by Red Hat and you will not be able to upgrade from a release candidate to
+the general availability release of {product-title}. Candidates should be used to
+test feature acceptance and assist in qualifying the next version of
+{product-title} in your infrastructure.
++                                                                                    
+[NOTE]                                                                               
+==== 
+Release candidates differ from the nightly builds found on try.openshift.com. You
+cannot upgrade nightly builds to nightly builds. Nightly builds are available for
+early access to features but are not upgradable or supported.
+==== 
+
+For GA versions, the fast and stable channels present a choice between receiving
+updates as soon as they are available or allowing Red Hat to control the rollout of
+those updates.
+
+= fast-{product-version}
+
+The fast channel is updated with new  {product-version}patch versions as soon as Red
```

Comment:
nit: shift a space.  You currently have two before `{product-version}` and none after.